src/store/apiSlice.js

```diff
@@ -41,7 +41,15 @@ export const apiSlice = createApi({
         return awsSources;
       },
     }),
+    getArchitecturesByDistribution: builder.query({
```

Comment:
Mostly because i don't really understand, what's the difference between adding it under `endpoints:` and adding it as a field directly? 

Reading https://redux-toolkit.js.org/rtk-query/api/createApi it seems they add all queries under `endpoints`.

Reply:
oh wait nvm, I can't read. It's added under endpoints :facepalm: